hashintel/hash
# Fix entity type editor table positioning in Safari

## 🌟 What is the purpose of this PR?

This PR fixes a visual bug happens on Safari, at the type editor.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1204315348455207/1204629004398918/f) _(internal)_


## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:













### 📜 Does this require a change to the docs?




The changes in this PR:







### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:







## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Check the type editor visually on Safari and Chrome. Safari should look ok.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204284519196810
